src/App.js

```diff
@@ -1,16 +1,57 @@
 import React from 'react';
+import TodoForm from './components/TodoComponents/TodoForm.js';
+import TodoList from './components/TodoComponents/TodoList.js';
+
 
 class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      todos: [],
+      task: "",
+      id: 545484,
+      completed: false
+    };
+  }
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+  handleSubmit = event => {
+    event.preventDefault();
+
+    this.setState({
+      toDoItems: [
+        ...this.state.todos,
+        {
+          task: this.state.task,
+          id: 545454,
+          completed: false 
```

Comment:
inside the setState here when you are building the new object, this is where you will want the Date.now() method to generate your new id. You will only hard code the id for any dummy/starter data you provide your app with